Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -30,6 +30,10 @@ inline model::FieldPath Field(absl::string_view field) {
   return model::FieldPath::FromServerFormat(field);
 }
 
+// Add a non-inline function to make this library buildable.
+// TODO(zxu123): remove once there is non-inline function.
```

Comment:
A quick note: cmake *does* support header only libraries, so we likely don't have to add this dummy function/file. However, if we expect this to *not* be a header only library in the long term, then adding a dummy function/file like you've done sounds like the correct approach.